reactive/etcd.py

```diff
@@ -154,6 +154,7 @@ def send_single_connection_details(db):
 
 
 @when('proxy.connected')
+@when('etcd.ssl.placed')
```

Comment:
It seems to make sense that we wait until the ssl is placed before we use it 3 lines down.